### ISSUE TITLE
[Proposal] Get Rid of Large Ship Time Dilation

### ISSUE THREAD

**Issue by bobbens** [naev/naev#2050]
Naev has a mechanic where small ships "slow down time" and large ships "dilate time" so that small ships don't feel ultra-fast and large ships don't feel ultra-slow. However, given the speed of combat and the fact that small ships are still able to damage large ships (vendetta with banshees or whatever), it doesn't seem like it works very well. It's been weakened several times, and I believe it is still too strong. Instead of weakening it to oblivion, it may be better to just get rid of it as a general mechanic. That said, it doesn't mean it can't still be implemented case-by-case with ship-stats for tiny ships like the hyena (where it actually works pretty well). As a side note, it'll help avoid the high pitch shift that all large ships get that is somewhat comical, and it will also simplify explanations to the player.

To summarize, I would propose:
1. Get rid of `time_mod` as a ship attribute
2. Add some `time_mod` to some ships where it worked well (mainly interceptors such as the hyena, and maybe the gawain)

**Comment by PhoenixRiver** [comment]
I might suggest that if ever a ship big enough (e.g. The Emporer's First) or a station (or larger) be playable by the player (doesn't necessarily need to be flyable) it would be worth considering adding this to them also... but it seems like it would be better an exception rather than a rule and for situations that are clearly going to handle differently to anything the player is going to work with early on in their career...
